x-pack/plugins/endpoint/public/embeddables/resolver/view/process_event_dot.tsx

```diff
@@ -383,9 +550,10 @@ export const ProcessEventDot = styled(
   border-radius: 10%;
   white-space: nowrap;
   will-change: left, top, width, height;
-  contain: strict;
+  contain: layout;
```

Comment:
:information_source: A recent article about this: https://css-tricks.com/lets-take-a-deep-dive-into-the-css-contain-property/

The reason it had to be changed to `layout` from `strict` is that the menu can now overflow the bounds of the container, so we can no longer guarantee `paint` containment. May be worth revisiting later if we encounter rendering  jank with Resolver.